### PR TITLE
FIX: add params verification 

### PR DESCRIPTION
**Pull request by fpChan**
add params verification to fix [issue#274](https://github.com/okex/okchain/issues/274)

- Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/okex/okchain/blob/develop/CONTRIBUTING.md#pr-targeting))

- [ ] Linked to github-issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [ ] Updated relevant documentation (`docs/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Reviewed `Files changed` in the github PR explorer

______

For Admin Use:
- Added appropriate labels to PR (ex. wip, ready-for-review, docs)
- Reviewers Assigned
- Squashed all commits, uses message "Merge PR #XYZ: [title]" ([coding standards](https://github.com/okex/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: x/dex/keeper/querier.go [hunk 1]
```diff
@@ -39,7 +39,9 @@ func queryProduct(ctx sdk.Context, req abci.RequestQuery, keeper IKeeper) (res [
 	if errUnmarshal != nil {
 		return nil, sdk.ErrUnknownRequest(sdk.AppendMsgToErr("incorrectly formatted request data", errUnmarshal.Error()))
 	}
-
+	if params.PerPage <= 0 || params.Page <= 0 {
+		return nil, sdk.ErrUnknownRequest(fmt.Sprintf("invalid params: %d or %d", params.Page, params.PerPage))
```
Comment (ilovers) [inline x/dex/keeper/querier.go:43]:
```suggestion
		return nil, sdk.ErrUnknownRequest(fmt.Sprintf("invalid params: page=%d or per_page=%d", params.Page, params.PerPage))
```

File: x/order/keeper/querier.go [hunk 2]
```diff
@@ -186,7 +189,9 @@ func queryDepthBookV2(ctx sdk.Context, path []string, req abci.RequestQuery, kee
 	if err != nil {
 		return nil, sdk.ErrUnknownRequest(err.Error())
 	}
-
+	if params.Size <= 0 {
+		return nil, sdk.ErrUnknownRequest(fmt.Sprintf("invalid params: %d", params.Size))
```
Comment (ilovers) [inline x/order/keeper/querier.go:193]:
```suggestion
		return nil, sdk.ErrUnknownRequest(fmt.Sprintf("invalid param: size= %d", params.Size))
```

File: x/dex/keeper/querier.go (hunk 1 above)
Comment (fpChan) [reply x/dex/keeper/querier.go]: done

File: x/order/keeper/querier.go (hunk 2 above)
Comment (fpChan) [reply x/order/keeper/querier.go]: done

File: x/dex/keeper/querier.go [hunk 3]
```diff
@@ -39,7 +39,9 @@ func queryProduct(ctx sdk.Context, req abci.RequestQuery, keeper IKeeper) (res [
 	if errUnmarshal != nil {
 		return nil, sdk.ErrUnknownRequest(sdk.AppendMsgToErr("incorrectly formatted request data", errUnmarshal.Error()))
 	}
-
+	if params.PerPage <= 0 || params.Page <= 0 {
```
Comment (zhongqiuwood) [inline x/dex/keeper/querier.go:42]: use uint

File: x/order/keeper/querier.go [hunk 4]
```diff
@@ -56,7 +56,7 @@ type QueryDepthBookParams struct {
 
 // NewQueryDepthBookParams creates a new instance of QueryProposalParams
 func NewQueryDepthBookParams(product string, size int) QueryDepthBookParams {
-	if size == 0 {
```
Comment (zhongqiuwood) [inline x/order/keeper/querier.go:59]: use uint

File: x/dex/keeper/querier.go (hunk 3 above)
Comment (fpChan) [reply x/dex/keeper/querier.go]: done

File: x/order/keeper/querier.go (hunk 4 above)
Comment (fpChan) [reply x/order/keeper/querier.go:59]: done

**Pull request merged**